Software_Carbon_Intensity/Appendix_A.md

```diff
@@ -1 +1,101 @@
+	2) Mobile app connecting to backend APIs on cloud or on-premise
+	3) Desktop app connecting to backend APIs on cloud or on-premise
+	4) AI based machine learning model experiments
+	5) Open Source or Closed Source Framework SDKs
+	6) Server less applications
+
+From a pure monitoring perspective of the SRE metric , doing it on server based workloads in the above list may be the first step . For e.g Web based multi-tier applications have either Virtual machines or EC2s connecting to APIs and databases and hence we can monitor the Operational emissions of these server components.  Similarly we can calculate the metric for backend server APIs and serverless components of the Mobile and desktop apps.
+
+There would be challenges however in doing the same for desktop devices and mobile platforms as the emissions calculations would need to know at a rough level the total number of  mobile devices or desktops , their types etc to calculate roughly the value for the desktop apps.   Hence for the first release of specifications, we can propose that monitoring will be for a subset of the above workloads i.e for workloads which have majorly server components.
+
+
+2) Hosting Infrastructure - Dedicated Vs Shared
+
+Monitoring techniques will also vary based on the hosting mode for the software. For those with dedicated infrastructure, SCI will just be the sum total of Operational Emission values across the different layers. In the equation for SCI ( SCI per unit of baseline = (E*I) + M , the value of M does not make an impact when calculating delta carbon intensity = Current CI- Original CI since the hardware is exclusively reserved for the purpose of the said software. Hence the monitoring technique can potentially look for variances in the Operational Emissions value and variances of it to raise alert for the Site Reliability Engineer.
+
+The situation is different when we consider Shared infrastructure servers, multi-tenant databases, SaaS software shared by multiple customers. Here multiple micro-services could share the same PaaS compute platforms and storage services which by design is carbon friendly. In these cases, the percentage of allocation of infrastructure  is necessary information to be able to calculate the carbon intensity value for the specific customer software. Hence we need to include the Embodied Emissions (M) value in the monitoring metric . 
+
+
+3) Application Environment Types
+The usage of the above SRE metric also changes by environment. This statement is tied to the Application architecture factor somewhat but broadly, the concept is that for the purpose of carbon tracking and monitoring, measurement should be done for all environments like development, testing, QA, Performance and Production. This is because the carbon emission of the software increases manifold for lower environments like development and QA for workloads like machine learning models. 
```

Comment:
I've clarified here that you are probably talking about "utilization" in terms of lower. Otherwise that was ambiguous. 

Also, in development, the carbon emissions for training ML models is actually quite high so I have removed that example from the last sentence. Please feel free to include another example @srini1978. Thanks!